shared/elements/gaia_grid/style.css

```diff
@@ -211,11 +211,17 @@
   height: 3rem;
 }
 
-.group .header span {
+.group .header span,
```

Comment:
We cannot remove span selector because there is an another span element (.gripper). It is purely decorative so that we cannot use button element for it.

I can move button specific styles to `.group .header .toggle` block